src/rust/engine/fs/Cargo.toml

```diff
@@ -13,7 +13,7 @@ digest = "0.6.2"
 futures = "^0.1.16"
 futures-cpupool = "0.1"
 glob = "0.2.11"
-grpcio = { version = "0.3", features = ["secure"] }
+grpcio = { git = "https://github.com/illicitonion/grpc-rs.git", rev = "d106c615bc0c289ba6d1ce6871786266d109c31c", features = ["secure"] }
```

Comment:
Comments indicating why we're forked would be good, but I guess not strictly necessary as long as we're able to look at the merge base.